modules/nw-disabling-hsts.adoc

```diff
@@ -8,7 +8,7 @@ To disable HTTP strict transport security (HSTS) per-route, you can set the `max
 
 .Prerequisites
 
-* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You are logged in to the cluster with a user with an admin privileges for the project.
```

Comment:
I think "with admin privileges" reads more naturally (omitting "an")

Reply:
@lcarva I have fixed it. Please check.